modules/git/repo_compare.go

```diff
@@ -25,6 +26,29 @@ type CompareInfo struct {
 	NumFiles  int
 }
 
+// DiffFiles slice of DiffFile
+type DiffFiles struct {
+	Files []*DiffFile
+}
+
+// DiffFile represents stats of a file modified between two commits (based on gitdiff.DiffFile)
```

Comment:
I would sort of like to move DiffFile type out of service/gitdiff and to here because I can't use it here due to circular dependency. I think ultimately GetDiffAllStats  could return Diff Type with all we need

Should be for a separate refactor after that doesn't also change features.

Reply:
The main problem is it depends on `models.Comment`.